sound/soc/sof/intel/Kconfig

```diff
@@ -150,6 +150,7 @@ config SND_SOC_SOF_CANNONLAKE_SUPPORT
 config SND_SOC_SOF_CANNONLAKE
 	tristate
 	select SND_SOC_SOF_HDA_COMMON
+	select SND_SOC_SOF_SOUNDWIRE
```

Comment:
Should we do this for SND_SOC_SOF_ICELAKE as well?

Reply:
yes indeed.